### DISCUSSION TITLE
20231218 오전 미팅 (10:00 ~ 21:30)

### DISCUSSION THREAD

**Discussion by Nahyun-Kang** [Codeit-Part3-Team3/Taskify#4] [Daily_Meeting]
#### 1. 깃헙 레포 생성 및 설정 완료

- project 만들기
- label 만들기
- branch protection rule 설정
- commit template 만들기
- issue template 만들기
- discussions template 만들기

#### 2. 프로젝트 생성 및 세팅

- Tailwind 적용 여부 : O **(적용)**
- 루트 폰트 사이즈 설정 10px vs 16px: **16px**
- 다운로드: **prettier, eslint, prettier plugin, tailwind css , recoil , axios, typescript**
- axios 사용 여부 : **O(적용)**
- Type vs interface : **interface**

### ❤️ 오늘 할일

- 깃헙 설정
- 프로젝트 생성 및 세팅
- R&R 분배
- Tailwind 설정 (Tailwind 적용시)

### ❤️ 내일 할 일

- 혜지님: 랜딩페이지
- 민혁님: input (드롭다운 바)
- 나현님: 사이드 메뉴
- 수영님: 칩, 버튼
- 대호님: input (댓글, 제목, 마감일, 태그)

**Comment by ngsy** [comment]
이 사람.. 완벽하다..!
